nifi-nar-bundles/nifi-hive-bundle/nifi-hive3-processors/src/main/java/org/apache/nifi/processors/hive/PutHive3Streaming.java

```diff
@@ -151,28 +151,19 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
-    static final PropertyDescriptor PARTITION_VALUES = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor STATIC_PARTITION_VALUES = new PropertyDescriptor.Builder()
             .name("hive3-stream-part-vals")
-            .displayName("Partition Values")
+            .displayName("Static Partition Values")
             .description("Specifies a comma-separated list of the values for the partition columns of the target table. If the incoming records all have the same values "
                     + "for the partition columns, those values can be entered here, resulting in a performance gain. If specified, this property will often contain "
                     + "Expression Language, for example if PartitionRecord is upstream and two partitions 'name' and 'age' are used, then this property can be set to "
-                    + "${name},${age}.")
+                    + "${name},${age}.\n"
+                    + "Having this property filled implies Hive Static Partitioning.")
```

Comment:
We should elaborate on this for clarity, as well as updating the `CapabilityDescription` above to say that if `Static Partition Values` is set, the partition columns (if present in the record) are ignored, and if `Static Partition Values` is not set, the partition columns must be declared at the end of the record in the order they were declared in the table definition. We can add similar text to both the Capability Description and this property, although the property doc can be less verbose.

Reply:
Hello, and sorry for the late reply. 

Thanks for the update and the tips. I'm working on it